### DISCUSSION TITLE
Convenience lost on right-click

### DISCUSSION THREAD

**Discussion by PedEchsing** [ViennaRSS/vienna-rss#1740] [General]
Don't know if this is programming error or deliberate changes but,
I am getting missing actions on the current ver. Vienna.
MacOS Sonoma 14.3.1 (23D60): Vienna Version 3.9.0 :7ad9df044: (8175)

I open an link in a new tab. Try to open a new link (from that tab) in a new tab with right-click, current page
goes to the the new link. Does NOT open in new tab. Opens in existing tab. HAVE to r-click the new link and select 'Open in new tab'.
Command-clicking doesn't work. Generally Option-clicking doesn't work (usually used for 'open in browser').

I CAN get around by r-clicking and selecting option, but no longer follows general browser conventions.

Is this going to be the default action going forward?

**Comment by barijaona** [comment]
Sorry, I missed your message.

I probably don't understand the issue…
To me, Vienna behaves like other browsers do on this matter (according to my tests with Safari and Firefox)

**Comment by PedEchsing** [comment]
It's been a long while since I sent the message OR logged into this account
to read messages...
I believe what I was writing about was; on Mac, using Vienna, when
command-clicking a link opens in a new tab.
When command-clicking a new link, IN THAT TAB, it SHOULD open in a NEW tab.
It doesn't.
It will open that new link in the same tab (yes you can still go back to
the previous page). Or you can force new tab by
right-clicking/control-clicking.

Sometimes when reading I'll open multiple tabs following multiple
links-of-interest in each tab. I LIKE being able to get back
to the original article by closing the other tab/links that I'd opened
instead of having to back-arrow to get to the original article.

Hopefully I'll be answered by a new update re-enabling this former action.

TRIVIA: I haven't logged into Github or this email account since I opened
the thread on GitHub. So if you reply, or ask questions that
need answers it will probably be weeks to months before I see replies.

**Comment by barijaona** [comment]
Code reimplementing command-click is currently under review (https://github.com/ViennaRSS/vienna-rss/pull/1803).
The functionality should be back in next version.
